backend.native/tests/build.gradle

```diff
@@ -3301,6 +3301,12 @@ linkTest("inline_defaultArgs_linkTest") {
     lib = "codegen/inline/defaultArgs_linkTest_lib.kt"
 }
 
+linkTest("inline_sharedVar_linkTest") {
```

Comment:
Does this test fail on JS? If not, how does JS backend handle this? Using permanent IR approach?

Reply:
They run all lowerings on the entire world.

Reply:
What about incremental compilation?

Reply:
For IC it is ensured that before a particular lowering is about to run all previous lowerings have been run.